lib/std.dl

```diff
@@ -14,6 +14,11 @@ extern function deref(x: Ref<'A>): 'A
 extern function max(x: 'A, y: 'A): 'A
 extern function min(x: 'A, y: 'A): 'A
 
+/*
+ * Arithmetic functions
+ */
+extern function pow32(base:'A, exp: bit<32>): 'A
```

Comment:
do you really expect we'll have other versions of pow?
Why don't you just call it pow?

Reply:
32 bits is sufficient, but because DDlog is strict with types and doesn't have type casting right now, it may be convenient to pass other integer types without conversion. Also, I cannot easily make this function polymorphic. So lets keep the name.